code/game/machinery/computer/crew.dm

```diff
@@ -79,9 +79,9 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 							datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
 	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
 	if (!ui)
-		var/datum/asset/simple/assets = get_asset_datum(/datum/asset/simple/mapimage)
-		assets.send(user)
-		ui = new(user, src, ui_key, "crew", "Crew Monitoring Console", 800, 600 , master_ui, state)
+		for(var/datum/minimap/M in SSmapping.station_minimaps)
```

Comment:
is this all minimaps for every station?

Reply:
no, it's all minimaps for the current station's z-levels, each z-level is its own minimap